dataRetriever/resolvers/epochproviders/arithmeticEpochProvider_test.go

```diff
@@ -64,9 +50,15 @@ func TestNewArithmeticEpochProvider_ShouldWork(t *testing.T) {
 	t.Parallel()
 
 	arg := ArgArithmeticEpochProvider{
-		RoundsPerEpoch:          2400,
-		RoundTimeInMilliseconds: 6000,
-		StartTime:               time.Now().Unix(),
+		ChainParametersHandler: &chainParameters.ChainParametersHandlerStub{
+			CurrentChainParametersCalled: func() config.ChainParametersByEpochConfig {
+				return config.ChainParametersByEpochConfig{
+					RoundDuration:  6000,
+					RoundsPerEpoch: 2400,
+				}
+			},
+		},
+		StartTime: time.Now().Unix(),
```

Comment:
Extract this in a func and use it in the other tests

Reply:
and add also a TODO to be handled by seconds or milliseconds (there is already an open PR with a common function for this)